servicepulse/redirect.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Redirects
+summary: Describes what message redirects are and how to use them.
+tags:
+- ServicePulse
+related:
+- servicepulse/troubleshooting
+---
+
+## Message Redirects
+
+In Versions **1.6.6** and above ServicePulse includes a screen to view and manage failed message redirects that have been created to send failed messages to an alternate queue when the original processing endpoint address is no longer available.
+
+INFO: Message redirects are only a feature of ServicePulse/ServiceControl and will not alter the routing for NServiceBus endpoints.
```

Comment:
I am not sure as well. I don't want anyone to look at it as a routing override. Let me change it for warning